bundles/org.openhab.binding.melcloud/src/main/resources/ESH-INF/thing/acDevice.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="melcloud"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="acdevice">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="melcloudaccount" />
+		</supported-bridge-type-refs>
+		<label>A.C. Device</label>
+		<description>Air conditioning device</description>
+
+		<channels>
+			<channel id="hasPendingCommand" typeId="hasPendingCommand-channel" />
+			<channel id="offline" typeId="offline-channel" />
+			<channel id="power" typeId="power-channel" />
+			<channel id="operationMode" typeId="operationMode-channel" />
+			<channel id="setTemperature" typeId="setTemperature-channel" />
+			<channel id="fanSpeed" typeId="fanSpeed-channel" />
+			<channel id="vaneHorizontal" typeId="vaneHorizontal-channel" />
+			<channel id="vaneVertical" typeId="vaneVertical-channel" />
+			<channel id="roomTemperature" typeId="roomTemperature-channel" />
+			<channel id="lastCommunication" typeId="lastCommunication-channel" />
+			<channel id="nextCommunication" typeId="nextCommunication-channel" />
+		</channels>
+
+		<config-description>
+			<parameter name="deviceID" type="integer" required="true">
+				<label>Device ID</label>
+				<description>Device ID of the A.C. device</description>
+			</parameter>
+			<parameter name="buildingID" type="integer">
+				<label>Building ID</label>
+				<description>Building ID of the A.C. device.</description>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" required="true">
+				<label>Polling Interval</label>
+				<description>Time interval how often poll data from MELCloud</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<!-- Channels types -->
+
+	<channel-type id="power-channel">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power status of device</description>
+	</channel-type>
+	<channel-type id="operationMode-channel">
+		<item-type>Number</item-type>
```

Comment:
When using state options the item-type must be `String`. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#state-description

Reply:
@Hilbrand just to make sure I understand.. this would also require handleCommand, updateState and items for these channels to be changed from Number to String, correct?

Reply:
Yes that is correct.